src/components/TextField.vue

```diff
@@ -400,6 +415,12 @@ export default {
     }
   }
 
+  &--required {
+    .text-field__required-text {
```

Comment:
Personally I don't think you need to nest this inside `&--required` at all. If the element is there, the styles for it should be there too. If it's not there then it doesn't matter. The parent modifier class doesn't seem particularly relevant to me.

Reply:
got it thanks will remove nesting but leave modifier?

Reply:
I think he's saying you can remove the modifier. It can just be `.text-field__required-text` because the "required text" style is applicable regardless of whether the field is actually required or not.

Reply:
The modifier might be useful at some point, I wasn't explicitly saying you should remove that, just the nesting of this definition